spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/dynatrace/DynatraceProperties.java

```diff
@@ -97,4 +139,36 @@ public void setGroup(String group) {
 		this.group = group;
 	}
 
+	public String getMetricKeyPrefix() {
+		return metricKeyPrefix;
+	}
+
+	public void setMetricKeyPrefix(String metricKeyPrefix) {
+		this.metricKeyPrefix = metricKeyPrefix;
+	}
+
+	public Boolean getEnrichWithOneAgentMetadata() {
+		return enrichWithOneAgentMetadata;
+	}
+
+	public void setEnrichWithOneAgentMetadata(Boolean enrichWithOneAgentMetadata) {
+		this.enrichWithOneAgentMetadata = enrichWithOneAgentMetadata;
+	}
+
+	public Map<String, String> getDefaultDimensions() {
```

Comment:
I'm not sure if it is the preferred way in spring boot, but I might make this immutable: `Collections.unmodifiableMap(defaultDimensions);` (I don't really remember if configprops will like it or not).

Reply:
It should be modifiable for consistency with other configuration properties classes.

Reply:
I reset it to be modifiable. Thanks for the clarification!